pkgs/by-name/tl/tlm/package.nix

```diff
@@ -0,0 +1,35 @@
+{
+  lib,
+  fetchFromGitHub,
+  buildGoModule,
+  unstableGitUpdater,
+  ollama,
+}:
+
+buildGoModule rec {
+  pname = "tlm";
+  version = "1.1";
+
+  src = fetchFromGitHub {
+    owner = "yusufcanb";
+    repo = "tlm";
+    rev = "${version}";
+    hash = "sha256-UKvgwn8yvjcREWhmwLIN8Vpbn8Ud/7wu+IluAXTVZRY=";
+  };
+  buildInputs = [ ollama ];
+  vendorHash = "sha256-hd/xj/PX0p7Ol0zan420QQzXbZLoDZRI1VQmcraeOTc=";
+
+  # Test expects a config file, so we create an empty one
+  checkPhase = ''
+    touch .tlm
```

Comment:
You are overriding the default test code, that would skip the test. Add it in `preCheck` instead.